assets/js/frontend/everest-forms.js

```diff
@@ -215,7 +234,51 @@ jQuery( function ( $ ) {
 						}
 
 						form.submit();
-					}
+					},
+					onkeyup: function( element, event ) {
+						// This code is copied from JQuery Validate 'onkeyup' method with only one change: 'everest-forms-novalidate-onkeyup' class check.
+						var excludedKeys = [ 16, 17, 18, 20, 35, 36, 37, 38, 39, 40, 45, 144, 225 ];
+
+						// Disable onkeyup validation for some elements (e.g. remote calls).
+						if ( $( element ).hasClass( 'everest-forms-novalidate-onkeyup' ) ) {
+							return;
+						}
+
+						if ( 9 === event.which && '' === this.elementValue( element ) || $.inArray( event.keyCode, excludedKeys ) !== -1 ) {
```

Comment:
Could use yoda condition on the 3rd expression within the conditional.

Reply:
Ref: https://github.com/wpeverest/everest-forms/pull/331/commits/56cb3f795db1e3bbc37726bfc852b143211e5a2b